aioseop_class.php

```diff
@@ -3661,11 +3661,10 @@ function wp_head() {
 		$description = '';
 		// Logging - rewrite handler check for output buffering.
 		$this->check_rewrite_handler();
-		if ( AIOSEOPPRO ) {
-			echo "\n<!-- All in One SEO Pack Pro $this->version by Michael Torbert of Semper Fi Web Design";
-		} else {
-			echo "\n<!-- All in One SEO Pack $this->version by Michael Torbert of Semper Fi Web Design";
-		}
+
+		echo "\n";
+		printf( '<!-- %1$s %2$s %3$s Michael Torbert %4$s Semper Fi Web Design', AIOSEOP_PLUGIN_NAME, $this->version, __( 'by', 'all-in-one-seo-pack' ), __( 'of', 'all-in-one-seo-pack' ) );
```

Comment:
This should probably be flipped, so that translators see the entire string, even if part of it is placeholders. 
Then we should use translator comments so they have context for the two words they're translating.